VideoLocker/src/main/java/org/edx/mobile/view/my_videos/MyRecentVideosFragment.java

```diff
@@ -67,19 +71,22 @@
     @Inject
     protected IEdxEnvironment environment;
 
+    @Inject
+    protected EventBus eventBus;
```

Comment:
Ah, I know what happened. The wrong EventBus was imported the first time I tried. Will fix.